### ISSUE TITLE
Topic Pool

### ISSUE THREAD

**Issue by zaemyung** [zaemyung/Seeking-SOTA#1]
N 세션을 잡아서 같이 파볼 주제들입니다. 자유롭게 추가 및 투표 해주세요!

**Comment by zaemyung** [comment]
### Graph Neural Networks
- From basic to recent papers
- Possible materials
  - https://www.youtube.com/watch?v=Iiv9R6BjxHM&list=PLvDlo2t54lUKMMDgNUPFGCHriT0bMBU91
  - Stanford CS224W - Machine Learning with graphs- Fall 2019, https://www.youtube.com/watch?v=uEPPnR22fxg&list=PL-Y8zK4dwCrQyASidb2mjj_itW2-YYx6-
  - [An Introduction to Graph Neural Network(GNN) For Analysing Structured Data](https://towardsdatascience.com/an-introduction-to-graph-neural-network-gnn-for-analysing-structured-data-afce79f4cfdc)
    - [(번역) GNN 소개 — 기초부터 논문까지](https://medium.com/watcha/gnn-%EC%86%8C%EA%B0%9C-%EA%B8%B0%EC%B4%88%EB%B6%80%ED%84%B0-%EB%85%BC%EB%AC%B8%EA%B9%8C%EC%A7%80-96567b783479)
  - (튜토리얼 페이퍼) A Gentle Introduction to Deep Learning for Graphs, https://arxiv.org/abs/1912.12693
  - (서베이 페이퍼) Graph Neural Networks: Taxonomy, Advances and Trends, http://arxiv.org/abs/2012.08752

**Comment by zaemyung** [comment]
### Computing Similarity between Trees/Graphs
- E.g. how similar are these two parse trees?

**Comment by zaemyung** [comment]
### Gumbel-Softmax
- The Gumbel-Softmax distribution is a continuous distribution that approximates samples from a categorical distribution and also works with backpropagation.
- What is Gumbel-Softmax?, https://towardsdatascience.com/what-is-gumbel-softmax-7f6d9cdcb90e
- Categorical Reparameterization with Gumbel-Softmax, https://arxiv.org/abs/1611.01144
- The Concrete Distribution: A Continuous Relaxation of Discrete Random Variables, https://arxiv.org/abs/1611.00712
- [ Python ] gumbel softmax 알아보기, https://data-newbie.tistory.com/263
